apps/frontend/src/style/theme.ts

```diff
@@ -274,21 +277,57 @@ export const theme: ThemeDefault = {
   radioRadius: 8,
   radioBorderRadius: 100,
   radioBorderColor: 'transparent',
-  radioButtonRadius: 4,
-  radioButtonBgdColor: themeColor.greyLight,
+  radioBnnRadius: 4,
+  radioBtnBgdColor: themeColor.greyLight,
 
   // BOX CONTAINER
   layoutsBoxContainerMargin: '40px 0',
-  layoutsBoxContainer: '20px 20px',
+  layoutsBoxContainer: '40px 20px',
   cardBoxContainer: '10px',
 
   // NAV
-  navBackgroundColor: themeColor.greyMedium,
+  navBackgroundColor: themeColor.secondaryColor,
 
   // CARD
   cardHeight: '20vh',
   cardWidth: '33vw',
 
   // LAYOUT
   layoutBGColor: themeColor.greyExtraLight,
+
+  // SIDEBAR PROJECT
+  sidebarProjectBG: themeColor.blueLight,
 };
+export const LabelStyle = styled.label`
```

Comment:
```suggestion
export const StyledLabel = styled.label`
```
Don't forgot to update usages as well.

Reply:
Old usage that has been forgotten